### PR TITLE
Modulesync 0.12.2 & Release 0.3.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
@roidelapluie can you release a 99.99.99 version of https://forge.puppet.com/roidelapluie/gerrit after this PR is done?

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## 2016-08-19 Release 0.3.0
+
+* Initial release in the VoX Pupuli namespace (module provided by roidelapluie)
+* Drop of ruby1.8.3
```
Comment (alexjfisher) [inline CHANGELOG.md]: Is this relevant for this module?
Module has no facts, functions or native types/providers.

Comment (bastelfreak) [inline CHANGELOG.md]: you're correct, fixed

**Comment by roidelapluie** [comment]
https://forge.puppet.com/roidelapluie/gerrit

**Pull request merged**